src/components/UserTableDialog.tsx

```
@@ -74,148 +107,189 @@ export function UserTableDialog({ selecionados, onChange }: Props) {
+                    <TableRow key={user.email}>
+                      <TableCell>
+                        <CustomCheckbox
+                          checked={marcados.has(user.email)}
+                          onCheckedChange={() => toggleMarcado(user.email)}
+                        />
+                      </TableCell>
+                      <TableCell>{user.nome}</TableCell>
+                      <TableCell>{user.email}</TableCell>
+                      {tabAtivo === 'candidatos' && (
+                        <TableCell>
+                          <span className={`px-2 py-1 rounded-full text-xs ${
```

ESLint

[error] Insert `⏎···························`

(prettier/prettier)